docs/int/faq/errors.mdx

```diff
@@ -386,4 +386,16 @@ docker compose logs
       </h4>
     </summary> The relay you are trying to connect to your peers via is offline or unreachable.
   </details>
+</details>
+<details open className="details">
+  <summary>
+    <h2 id="Lodestar">Lodestar</h2>
+  </summary>
+  <details className="details">
+    <summary>
+      <h4 id="Potential-next-epoch-attester-duties-reorg-slot-821-error">
+        <code> warn: Potential next epoch attester duties reorg</code>{" "} error
+      </h4>
+    </summary> Lodestar logs these warnings because charon is not able to return proper <code>dependent_root</code> value in <code>getAttesterDuties</code> API response whenever lodestar calls this API. This is because charon uses <code>go-eth2-client</code> for all the beacon API calls and it doesn't provide <code>dependent_root</code> value in responses. We have reported this to them here.
```

Comment:
```suggestion
    </summary> Lodestar logs these warnings because charon is not able to return proper <code>dependent_root</code> value in <code>getAttesterDuties</code> API response whenever lodestar calls this API. This is because charon uses <code>go-eth2-client</code> for all the beacon API calls and it doesn't provide <code>dependent_root</code> value in responses. We have reported this to them here.
```

"here" in this instance is not a URL, not going to merge this before we cut the latest release FYI.

Reply:
@dB2510 can you take a look at this one please

Reply:
@maeliosakyne you need to add this link at the end of the text what @OisinKyne for word "here": https://github.com/attestantio/go-eth2-client/issues/35